extract_data.py

```diff
@@ -17,89 +36,104 @@
 except FileExistsError:
     pass
 
-boxre = [xmin, xmax, zmin, zmax]
-
 for run in runs:
 
     run_id = run['id']
     start_time = run['t_min']
     end_time = run['t_max']
 
```

Comment:
Looks nicer if you have a consistent spacing on both sides of the equal sign. Such as `Pod_name = 'proton/vg_ptensor_offdiagonal'`